packages/block-library/src/block/edit.js

```diff
@@ -213,6 +215,7 @@ function ReusableBlockEdit( {
 		onNavigateToEntityRecord( {
 			postId: ref,
 			postType: 'wp_block',
+			selectedBlockClientId,
```

Comment:
Instead of having users of `onNavigateToEntityRecord` passing the selectedBlockClientId all the time. What if we augmented `onNavigateToEntityRecord` when we receive it in the `settings` prop of `BlockEditorProvider`. That way the consumers wouldn't have to care about doing this all the time. I'm 80% convinced about my own suggestion though, so let me know what you think haha